### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by JuanitoDeLaTorre**

### CODE REVIEW

**Review by alicolaktv — APPROVED** [review]

File: routes/pokemon2.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const express = require("express")
+
+const app = express()
+
+app.get('/', (req,res) => {
+    res.send("SWAG")
```
Comment (joshuapent) [inline routes/pokemon2.js]: This is phenomenal work. You should get a raise asap.

**Review by joshuapent — CHANGES_REQUESTED** [review]
noice.

File: routes/pokemon2.js (hunk 1 above)
Comment (JuanitoDeLaTorre) [reply routes/pokemon2.js]: I thought it was fine *eye roll*

**Comment by alicolaktv** [comment]
WOAH, I COULD NEVER!

**Review by joshuapent — APPROVED** [review]
You did it. kudos

**Pull request merged**